cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -2093,6 +2093,14 @@ spec:
                         type: string
                       startTime:
                         type: string
+            quotas:
```

Comment:
ah, missing the `type` and `nullable` fields

Reply:
Fixed.